docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,97 @@
+- **get_HVAC_building_area_types_and_zones**
+- **does_zone_meet_G3_1_1b**
+- **does_zone_meet_G3_1_1c**
+- **does_zone_meet_G3_1_1e**
+- **does_zone_meet_G3_1_1f**
+- **does_zone_meet_G3_1_1g**
+- **get_predominant_HVAC_building_area_type**
+- **get_number_of_floors**
+- **expected_system_type_from_Table_G3_1_1**
+
+**Data Lookup**
+- **Table G3.1.1**
+
+## Logic:
+- get climate zone (not sure this is the correct path): `climate_zone = ASHRAE229.weather.climate_zone`
+- create zones_and_systems list: `zones_and_systems = {}`
+- pre-fill the list with an empty string:
+	- `for building_segment in RMR.building.building_segments:`
+		- `for zone in building_segment.zones:`
+			- `zones_and_systems[zone] = []`
+- get the list of HVAC building area types and zones: `list_building_area_types_and_zones = get_HVAC_building_area_types_and_zones(B-RMR)`
+- get the predominant building area type: `predominant_building_area_type = get_predominant_HVAC_building_area_type(list_building_area_types_and_zones)`
+- get the total number of floors for the building: `num_floors = get_number_of_floors(B-RMR)`
+- first get the expected system type for the predominant building area type.  To do this, we need the total area of the predominant building area type, which includes all BAT's less than 20000 ft2
+- set area equal to the area of the predominant building area type: `area = list_building_area_types_and_zones[predominant_building_area_type]["AREA"]`
+- loop through the building area types: `for bat in list_building_area_types_and_zones:`
+	- if the bat is not equal to the predominant building area type: `if bat != predominant_building_area_type:`
+		- and if the area of this bat < 20,000 ft2: `if list_building_area_types_and_zones[bat]["AREA"] <= 20000:`
+			- add this area to the area value for determining predominant HVAC system type: `area = area + list_building_area_types_and_zones[bat]["AREA"]`
+- expected_system_type_list = expected_system_type_from_Table_G3_1_1(predominant_building_area_type,num_floors,area,climate_zone)
+- fill the zones_and_systems list with the expected system and description string for all zones (we'll overwrite the other zones in the next step): `for zone in zones_and_systems:`
+	- set the value to expected_system_type_list: `zones_and_systems[zone] = expected_system_type_list`
+	
+- now we need to go through each 'exception' to Table_G3_1_1 in order.
+
+- G3.1.1b "Use additional system types for nonpredominant conditions (i.e., residential/nonresidential) if those conditions apply to more than 20,000 ft2 of conditioned floor area." uses the same logic that we just used for the expected_system_type_list, except that we select systems based on non-predominant building area types and the area of these non-predominant conditions
+- loop throught the building area types: `for bat in list_building_area_types_and_zones:`
+	- look for any bat's that have more than 20000 ft2 area: `if list_building_area_types_and_zones[bat]["AREA"] >=20000:`
```

Comment:
do we need to put if-skip to the predominant BAT here?

Reply:
yes.  thanks